src/routes/docs/tooling/command-center/+layout.svelte

```diff
@@ -1,9 +1,32 @@
 <script lang="ts">
 	import Docs from '$lib/layouts/Docs.svelte';
-	import Sidebar from '../../Sidebar.svelte';
+	import Sidebar, { type NavParent, type NavTree } from '$lib/layouts/Sidebar.svelte';
+
+	const parent: NavParent = {
+		href: '/docs',
+		label: 'Command Center'
+	};
+
+	const navigation: NavTree = [
+		{
+			label: 'Guides',
+			items: [
+				{
+					label: 'Command Center',
+					href: '/docs/tooling/command-center/command-center'
+				},{
```

Comment:
Did the formatter put the braces on the same line like this? If not, please change it so it matches the others.